app/page-partials/my-dashboard/RoleManagement/RoleEnvironment.tsx

```diff
@@ -311,129 +313,122 @@ const RoleEnvironment = ({ environment, integration, alert }: Props) => {
+
+            const [guid, idp] = usernameSplit;
+            const idpMeta = propertyOptionMap[idp];
+
+            return {
+              idp: idpMap[idp],
+              guid: guid,
+              email: user.email,
+              actions: (
+                <>
+                  <span
+                    onClick={(event) => {
+                      event.stopPropagation();
+
+                      infoModalRef.current.open({
+                        guid: user.username.split('@')[0],
+                        attributes: {
+                          ...reduce(
+                            idpMeta,
+                            (ret: { [key: string]: string }, keyVal: PropertyOption) => {
+                              ret[keyVal.label] = get(user, keyVal.value);
+                              return ret;
                             },
-                          });
-                        }}
-                      >
-                        <FontAwesomeIcon style={{ color: '#000' }} icon={faEye} size="lg" title="User Detail" />
-                      </span>
-                      &nbsp;&nbsp;
-                      <span onClick={() => removeUserModalRef.current.open(user)}>
-                        <FontAwesomeIcon
-                          style={{ color: '#FF0303' }}
-                          icon={faMinusCircle}
-                          size="lg"
-                          title="Remove User"
-                        />
-                      </span>
-                    </td>
-                  </tr>
-                );
-              })}
-            </InfScroll>
-          ) : (
-            <tbody>
-              <tr>
-                <td colSpan={5}>No users found.</td>
-              </tr>
-            </tbody>
-          )}
-        </Table>
+                            {},
+                          ),
+                          ...user.attributes,
+                        },
+                      });
+                    }}
+                  >
+                    <FontAwesomeIcon style={{ color: '#000' }} icon={faEye} size="lg" title="User Detail" />
+                  </span>
+                  &nbsp;&nbsp;
+                  <span onClick={() => removeUserModalRef.current.open(user)}>
+                    <FontAwesomeIcon style={{ color: '#FF0303' }} icon={faMinusCircle} size="lg" title="Remove User" />
+                  </span>
+                </>
+              ),
+            };
+          })}
+          colfilters={[]}
+          activateRow={noop}
+          rowSelectorKey={'guid'}
+          noDataFoundElement={<td colSpan={5}>No users found.</td>}
+        />
       );
     } else if (rightPanelTab === 'Service Accounts') {
       rightPanel = (
-        <Table variant="mini">
-          <thead>
-            <tr>
-              <th>Project Name</th>
-              <th className="text-right">Actions</th>
-            </tr>
-          </thead>
-          {filterServiceAccountUsers(users).length > 0 ? (
```

Comment:
why was this removed? (both filterService Accountusers and the infscroll

Reply:
logically it provides the duplicated filtering behavior, but added